packages/KSegmentedControl/KSegmentedControl.vue

```diff
@@ -68,24 +68,41 @@ export default {
   }
 }
 </script>
-<style scoped>
+<style lang="scss" scoped>
 .segmented-control .k-button {
   border-radius: 0;
   margin-left: -1px;
   flex: 1;
-}
-.segmented-control .k-button:focus {
-  z-index: 1;
-}
-.segmented-control .k-button:first-child {
-  border-radius: 3px 0 0 3px;
-  margin-left: 0;
-}
-.segmented-control .k-button:last-child {
-  border-radius: 0 3px 3px 0;
-}
-.segmented-control .k-button:only-child {
-  border-radius: 3px;
-  margin-left: 0;
+  color: var(--blue-600);
+  --KButtonPrimaryBase: var(--blue-100);
```

Comment:
Let's move these to the top of block. And then lets use a new var with a fallback of blue-600 and import the vars so like in other kongponents.

```scss
@import '~@kongponents/styles/_variables.scss';

.segmented-control .k-button {
--KButtonPrimaryBase: var(--blue-100);
--KButtonPrimaryHover: var(--blue-100);

color: var(--KSegementedControlPrimary, var(--blue-600, color(blue-600)));
```